extension/content/style.css

```diff
@@ -11,37 +11,56 @@ html, body {
 
 body {
   margin: 40px;
-  display: grid;
-  grid-template-areas:
-      "actionbox  actionbox  settings"
-      "polling    status     status"
-      "history    status     status"
-      "empty      status     status";
-  grid-template-columns: 1fr 2fr max-content;
-  grid-gap: 10px;
-
   color: var(--in-content-text-color);
 }
 
 body.loading {
   opacity: 0.5;
 }
 
-#action-box { grid-area: actionbox }
-#settings { grid-area: settings }
-#polling { grid-area: polling }
-#sync-history { grid-area: history }
-#status { grid-area: status }
+#header button {
+  display: inline-block;
```

Comment:
AFAIK inline block is when you want to have elements next to each other horizontally. That doesn't seem needed for our header buttons